### PR TITLE
Naver & Loyalty | New Icons

### PR DESCRIPTION
**Pull request by ZilvinasMaciulis**

### CODE REVIEW

File: packages/wix-ui-icons-common/src/general/metadata.ts [hunk 1]
```diff
@@ -31,6 +31,13 @@ const generalIconsMetadata: Array<IconMetadata> = [
     tags: ["Subscription", "Pricing", "Membership", "Members"],
     sizes: { "18": "PaidPlansSmall", "24": "PaidPlans" },
   },
+	{
+	 title: "Loyalty",
+	 category: "Wix Brands",
+	 description: "Button - opens Loyalty page",
+	 tags: ["Customer", "Coin", "Discount", "Points"],
+	 sizes: { "18": "LoyaltySmall”, "24": "Loyalty" },
+	 },
```
Comment (Zivilekin) [inline packages/wix-ui-icons-common/src/general/metadata.ts:40]: Could you align this block of code to the left?

File: packages/wix-ui-icons-common/src/general/metadata.ts [hunk 2]
```diff
@@ -2058,6 +2065,13 @@ sizes: { "18": "ESignatureSmall", "24": "ESignature" },
     tags: [ "share", "video", "upload"],
     sizes: { "18": "VimeoSmall", "24": "Vimeo" },
   },
+	{
+	 title: "Naver",
+	 category: "Brands",
+	 description: "Button - redirects to Naver browser",
+	 tags: ["browser", "search"],
+	 sizes: { "18": "NaverSmall", "24": "Naver" },
+	 },
```
Comment (Zivilekin) [inline packages/wix-ui-icons-common/src/general/metadata.ts:2074]: And this also align to the left

File: packages/wix-ui-icons-common/src/general/metadata.ts (hunk 1 above)
Comment (ZilvinasMaciulis) [reply packages/wix-ui-icons-common/src/general/metadata.ts]: @Zivilekin done

File: packages/wix-ui-icons-common/src/general/metadata.ts (hunk 2 above)
Comment (ZilvinasMaciulis) [reply packages/wix-ui-icons-common/src/general/metadata.ts]: @Zivilekin Done

**Review by Zivilekin — APPROVED** [review]

File: packages/wix-ui-icons-common/src/general/metadata.ts [hunk 3]
```diff
@@ -31,6 +31,13 @@ const generalIconsMetadata: Array<IconMetadata> = [
     tags: ["Subscription", "Pricing", "Membership", "Members"],
     sizes: { "18": "PaidPlansSmall", "24": "PaidPlans" },
   },
+  {
+  title: "Loyalty",
+  category: "Wix Brands",
+  description: "Button - opens Loyalty page",
+  tags: ["Customer", "Coin", "Discount", "Points"],
+  sizes: { "18": "LoyaltySmall”, "24": "Loyalty" },
```
Comment (Zivilekin) [inline packages/wix-ui-icons-common/src/general/metadata.ts:39]:
```suggestion
  sizes: { "18": "LoyaltySmall", "24": "Loyalty" },
```

**Review by Zivilekin — CHANGES_REQUESTED** [review]

**Review by Zivilekin — APPROVED** [review]

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-icons-common**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-icons-common.pr_2107/

**Pull request merged**